velox/experimental/wave/common/CMakeLists.txt

```diff
@@ -16,13 +16,19 @@ velox_add_library(
   velox_wave_common
   GpuArena.cpp
   Buffer.cpp
+  Compile.cu
   Cuda.cu
   Exception.cpp
+  KernelCache.cpp
   Type.cpp
   ResultStaging.cpp)
 
-velox_link_libraries(velox_wave_common velox_exception velox_common_base
-                     velox_type)
+velox_link_libraries(
+  velox_wave_common
+  velox_exception
+  velox_common_base
+  velox_type
+  CUDA::nvrtc)
```

Comment:
It seems to work without it but it probably makes sense to explicitly link to the runtime aswell via `CUDA::cudart`